RFCs/FS-1126-allow-lower-case-du-cases when-require-qualified-access-is specified.md

```diff
@@ -28,6 +28,9 @@ type DU = | a // error FS0053: Discriminated union cases and exception labels mu
 
 type DU = | a // error FS0053: Discriminated union cases and exception labels must be uppercase identifiers
```

Comment:
These two lines are identical and below there’s something similar with one with and one without `RequireQualifiedAccess`, but no explanation of the difference.